src/ensembl/production/metadata/api/models/assembly.py

```diff
@@ -59,6 +59,8 @@ class AssemblySequence(Base):
     md5 = Column(String(32))
     # column need renaming as well
     sha512t24u = Column(String(128))
+    type = Column(String(128), nullable=False)
```

Comment:
Is there any value in this being an enum so as to catch anything unexpected ?

Reply:
Fair point! Although a question for GBuilders to know what kind of value we can expect? @leannehaggerty  ?

Reply:
That is a fair point. I will do a bit of digging and see if we are actually consistent in our cores. If so, I would say definitly, but if we have to modify every second core to change Chromosome to chromosome, it may not be.

Reply:
As mentioned in Slack, I believe this restriction (e.g. Enum) can be put in place after this is merged. once we have a fair sample.